helpdesk-bot/src/main/java/org/symphonyoss/symphony/bots/helpdesk/bot/HelpDeskServiceConfiguration.java

```diff
@@ -96,6 +105,11 @@ public SymphonyValidationUtil getValidationUtil(SymphonyClient symphonyClient) {
     return new SymphonyValidationUtil(symphonyClient);
   }
 
+  @Bean(name = "IdleTimerManager", destroyMethod = "shutdown")
```

Comment:
The component name might start with lowercase character.

Reply:
ok